### PR TITLE
Update migration guide

### PR DESCRIPTION
**Pull request by markwinter**
- Update migration guide with more explanation
- Include additional steps for kubeflow-based kfserving migration
- I linked to the YAML in `kserve/kserve` repo instead of keeping a copy in this docs repo

Ref: https://github.com/kserve/kserve/pull/1873

### CODE REVIEW

**Comment by markwinter** [comment]
Hi @yuzisun can you review this when you have time please?

File: docs/admin/migration.md [hunk 1]
```diff
@@ -1,17 +1,81 @@
...
+    The migration job will by default delete the leftover KFServing installation after migrating the inference services from
+    `serving.kubeflow.org` to `serving.kserve.io`.
 
-!!! note
-    The migration job runs in kserve namespace. please make sure kserve running in the cluster.
 
+### Migrating from standalone KFServing
 
-### Migration
+1. Install KServe v0.7 using the [install YAML](https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve.yaml)
```
Comment (yuzisun) [inline docs/admin/migration.md:12]:
```suggestion
1. Install KServe v0.7 using the [install YAML](https://github.com/kserve/kserve/releases/download/v0.7.0/kserve.yaml)
```

File: docs/admin/migration.md [hunk 2]
```diff
@@ -1,17 +1,81 @@
...
+    - This will not affect existing services yet.
 
-Please use this [YAML file](./kserve_migration_job.yaml) to migrate existing inference services.
+    ```bash
+    kubectl apply -f https://github.com/kserve/kserve/blob/master/install/v0.7.0/kserve.yaml
```
Comment (yuzisun) [inline docs/admin/migration.md:16]:
```suggestion
    kubectl apply -f https://github.com/kserve/kserve/releases/download/v0.7.0/kserve.yaml
```

File: docs/admin/migration.md [hunk 3]
```diff
@@ -1,17 +1,81 @@
...
+3. Clean up the migration resources
+
+    ```bash
+    kubectl delete ClusterRoleBinding cluster-migration-rolebinding
+    kubectl delete ClusterRole cluster-migration-role
+    kubectl delete ServiceAccount cluster-migration-svcaccount -n kubeflow 
+    ```
+
+4. Update the models web app to use the new InferenceService API group `serving.kserve.io`
+    - Change the deployment image to `kserve/models-web-app:v0.7.0-rc0`
```
Comment (yuzisun) [inline docs/admin/migration.md:68]: The model webapp does not support the new api group yet right?

Comment (markwinter) [reply docs/admin/migration.md:68]: The web app uses the new api group already
https://github.com/kserve/models-web-app/blob/master/backend/apps/common/versions.py#L25

**Review by yuzisun — APPROVED** [review]

**Pull request merged**